APCAppCore/APCAppCore/UI/Onboarding/APCStudyOverviewCollectionViewController.m

```diff
@@ -181,6 +181,7 @@ - (void)setUpAppearance
     self.dateRangeLabel.textColor = [UIColor appSecondaryColor3];
     
     self.btnAlreadyParticipated.tintColor = [UIColor appPrimaryColor];
+    self.view.backgroundColor = [UIColor appPrimaryColor];
```

Comment:
yep want to make sure the study overview uses the app color and not whatever near-miss is hardcoded in the storyboard